docker-bake.hcl

```diff
@@ -62,7 +62,7 @@ target "runc-azlinux" {
         "DALEC_DISABLE_DIFF_MERGE" = DALEC_DISABLE_DIFF_MERGE
     }
     contexts = {
-      "dalec_frontend" = "target:frontend"
+        "dalec_frontend" = "target:frontend"
```

Comment:
Weird that the spacing was messed up.

Reply:
just some nit to move spacing at same level as the other objects in this file